docs/docs/chitchat-faqs.mdx

```diff
@@ -4,258 +4,78 @@ sidebar_label: Chitchat and FAQs
-These assistants allow the user to ask a simple question and get a response. We're going to
-build a basic FAQ assistant using features of Rasa designed specifically for this type of assistant.
 
-In this section we're going to cover the following topics:
+This page is a guide on the concepts and training data you need to handle non-contextual questions like
+FAQ and chitchat.
+You can use the example configuration and training data on this page to build an assistant
+that can handle a conversation like the one above, 
+or you can apply the guidelines to your own use case.
 
-* [Responding to simple intents](./chitchat-faqs.mdx#respond-with-memoization-policy) with the MemoizationPolicy
+## Concepts
 
-* [Handling FAQs](./chitchat-faqs.mdx#faqs-response-selector) using the ResponseSelector
+To handle FAQ and chitchat you'll need a rule-based dialogue management policy ([the `RulePolicy`](#rulepolicy)), and an easy way
+to return the appropriate response for a question ([the `ResponseSelector`](#retrieval-actions-and-responseselector)).  
 
-We're going to use content from [Sara](https://github.com/RasaHQ/rasa-demo), the Rasa
-assistant that, amongst other things, helps the user get started with the Rasa products.
-You should first install Rasa using the [Step-by-step Installation Guide](./installation.mdx#step-by-step-installation-guide)
-and then [prototype an assistant](./prototype-an-assistant.mdx)
-to make sure you know the basics.
+### `RulePolicy`
 
-To prepare for this tutorial, we're going to create a new directory and start a
-new Rasa project.
-
-```bash
-mkdir rasa-assistant
-rasa init
-```
-
-Let's remove the default content from this bot, so that the `data/nlu.md`, `data/stories.md`
-and `domain.yml` files are empty.
-
-<a aria-hidden="true" tabIndex="-1" className="anchor enhancedAnchor" id="respond-with-memoization-policy"></a>
-
-## Memoization Policy
-
-The MemoizationPolicy remembers examples from training stories for up to a `max_history`
-of turns. The number of “turns” includes messages the user sent, and actions the
-assistant performed. For the purpose of a simple, context-less FAQ bot, we only need
-to pay attention to the last message the user sent, and therefore we'll set that to `1`.
-
-You can do this by editing your `config.yml` file as follows (you can remove `TEDPolicy` for now):
+For FAQ and chitchat, you always want the same response (or action) for the same question. 
+[Rules](rules.mdx) allow you to do exactly that.
+To use rules, the you need to add the [`RulePolicy`](#rule-policy) to your policies in your configuration file:
 
 ```yaml
 policies:
-- name: MemoizationPolicy
-  max_history: 1
-- name: MappingPolicy
+- name: RulePolicy
 ```
 
-:::note
-The MappingPolicy is there because it handles the logic of the `/restart` intent,
-which allows you to clear the conversation history and start fresh.
-
-:::
+You can have other policies as well, but the `RulePolicy` is the only one necessary for handling rules.
```

Comment:
"can have other policies" seems a bit vague. Maybe a bit more context here?

The RulePolicy works alongside other policies in your configuration (like the TED Policy). Its job is to enforce rule-based logic for conversation paths where you already know how the assistant should respond to a given input.